package.json

```diff
@@ -21,7 +21,7 @@
     "test": "./node_modules/.bin/karma start --single-run",
     "test:dev": "karma start karma.conf.js",
     "storybook": "start-storybook -p 9001",
-    "storybook-preact": "REACT_IMPL=preact-compat npm run storybook"
+    "storybook:preact": "REACT_IMPL=preact start-storybook -p 9001"
```

Comment:
Can we keep `npm run storybook` here so that it would be enough to change the storybook CLI params in one place when needed?